src/game/camera.c

```diff
@@ -2875,6 +2974,11 @@ void update_camera(struct Camera *c) {
     OSTime first   = osGetTime();
     OSTime colTime = collisionTime[perfIteration];
 #endif
+
+#ifdef REONUCAM
+    extern s16 s8DirModeBaseYaw;
+#endif
```

Comment:
are you sure this is needed? i thought this was already declared at the top

Reply:
lmao yeah that makes no sense